mcux/drivers/imx/fsl_lpspi.c

```diff
@@ -1198,6 +1198,7 @@ static void LPSPI_MasterTransferComplete(LPSPI_Type *base, lpspi_master_handle_t
 {
     assert(handle);
 
+    LPSPI_ClearStatusFlags(base, kLPSPI_TransferCompleteFlag);
```

Comment:
I'm not sure this is the right fix. Can you look at updating `spi_mcux_master_transfer_callback` to parse the `status` argument instead?

If it is the right fix, it should be in a separate commit from the uint32_t overflow change.

Reply:
Thanks for your review.
clean flag can be put in spi_mcux_master_transfer_callback, when status == kStatus_Success. But, I saw in LPSPI_SlaveTransferHandleIRQ in HAL NXP that the clear interrupt flag is being processed. I think that the clear interrupt flag processing of Master transmission should also be put in HAL NXP.

Reply:
In this PR, the interrupted modification is removed and only the problem of time delay is dealt with. When the interrupted modification method is confirmed, I submit another commit.

Reply:
Thank you. Will you please also update the patch list in `mcux/README` to mention the delay change, and then I'll approve this PR.

Reply:
I have update the patch list, Please review again, Thanks